pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1715,6 +1715,13 @@ self: super: {
   # Issue reported upstream, no bug tracker url yet.
   darcs = doJailbreak super.darcs;
 
+  # Too strict version bounds on ansi-terminal;
+  colourista = doJailbreak (appendPatch super.colourista
```

Comment:
With the patch, jailbreak shouldn't be necessary (or rather the patch is kinda pointless).